redirector/common.py

```diff
@@ -80,13 +80,26 @@ def get_ddb_table():
     return boto3.resource("dynamodb").Table(MANIFEST_TABLE_NAME)
 
 
+def get_file_pathname(file):
+    # want to start the path after "All Files/<BoxFolderName>/"
+    file_path_collection = file.path_collection
+    file_path_names = [fp.get().name for fp in file_path_collection["entries"][2:]]
+    if len(file_path_names) > 0:
```

Comment:
This is a matter of taste, but it's possible to simplify this block by including the filename in the file_path_names array:

```python
file_path_names = [fp.get().name for fp in file_path_collection["entries"][start_index:] + [file.name]
return "/".join(file_path_names)
```

You may or may not find that easier to read, feel free to ignore this comment.

Reply:
This was implemented.